addons/l10n_cz/data/template/account.tax-cz.csv

```diff
@@ -1,39 +1,71 @@
-"","","","","","","","","","tax","invoice","",""
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","",""
-"vsc_nad_eu","Pořízení z EU","","21%","21.0","percent","purchase","","tax_group_vat_21","base","invoice","",""
-"","","","","","","","","","tax","invoice","chart_cz_343121",""
-"","","","","","","","","","tax","invoice","chart_cz_343221","-100"
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","chart_cz_343121",""
-"","","","","","","","","","tax","refund","chart_cz_343221","-100"
+"id","name","invoice_label","tax_scope","type_tax_use","amount","invoice_repartition_line_ids/factor_percent","invoice_repartition_line_ids/repartition_type","invoice_repartition_line_ids/document_type","invoice_repartition_line_ids/factor","invoice_repartition_line_ids/account_id","invoice_repartition_line_ids/tag_ids","refund_repartition_line_ids/factor_percent","refund_repartition_line_ids/repartition_type","refund_repartition_line_ids/document_type","refund_repartition_line_ids/factor","refund_repartition_line_ids/account_id","refund_repartition_line_ids/tag_ids","price_include","amount_type","tax_group_id"
+"l10n_cz_other_transaction_deduction","Other transactions with tax deduction","The transaction is exempt from VAT pursuant to 24a, 67, 68, 69 and 70, 89, 90, 92 of the Value Added Tax Act with the right to deduct the tax.","","sale","0","100","base","invoice","1","","+VAT 26","100","base","refund","1","","-VAT 26","False","percent","tax_group_vat_0"
+"","","","","","","100","tax","invoice","1","","","100","tax","refund","1","","","","",""
+"l10n_cz_12_purchase_goods_eu","12% - Purchase of goods from another EU country","12% VAT","consu","purchase","12","100","base","invoice","1","","+VAT 4 Base","100","base","refund","1","","-VAT 4 Base","False","percent","tax_group_vat_12"
+"","","","","","","100","tax","invoice","1","chart_cz_343112","","100","tax","refund","1","chart_cz_343112","-VAT 4 Tax","","",""
+"","","","","","","-100","tax","invoice","-1","chart_cz_343212","-VAT 4 Tax","-100","tax","refund","-1","chart_cz_343212","","","",""
+"l10n_cz_supply_goods_eu","Supply of goods to another EU country","The tax is paid by the customer","consu","sale","0","100","base","invoice","1","","+VAT 20","100","base","refund","1","","-VAT 20","False","percent","tax_group_vat_0"
+"","","","","","","100","tax","invoice","1","","","100","tax","refund","1","","","","",""
+"l10n_cz_21_acquisition_goods_eu","21% - Acquisition of goods from another EU country","21% VAT","consu","purchase","21","100","base","invoice","1","","+VAT 3 Base","100","base","refund","1","","-VAT 3 Base","False","percent","tax_group_vat_21"
+"","","","","","","100","tax","invoice","1","chart_cz_343121","","100","tax","refund","1","chart_cz_343121","-VAT 3 Tax","","",""
+"","","","","","","-100","tax","invoice","-1","chart_cz_343221","-VAT 3 Tax","-100","tax","refund","-1","chart_cz_343221","","","",""
```

Comment:
Multiple taxes where the total tax is null cause you do 100% - 100% 
![image](https://github.com/odoo/odoo/assets/109144554/e282adac-d7e4-4525-86fb-07b468da5a00)

So if we do a move with this tax, the taxes result is 0 :smile: Care with the taxes grid when you do the change
It happens in other taxes so check those one by one :smile:

Reply:
The PO says is should be fine :sweat_smile: